addon/index.js

```diff
@@ -54,14 +54,8 @@ export default Mixin.create({
   updateScrollPosition(transitions) {
     const lastTransition = transitions[transitions.length - 1];
```

Comment:
@rondale-sc minor eslint error.

Reply:
👌

Reply:
Sorry I haven't been able to keep on top of this properly.  My branch locally was green, but when I rebased master this morning I noticed the eslint error.  Should be gtg now.  Sorry again.

Reply:
@rondale-sc not at all! The issue was because of another package update—not your work. Thank you for making the change!